frontend_tests/node_tests/integration_bot_widget.js

```diff
@@ -0,0 +1,111 @@
+set_global('page_params', {
+    is_admin: false,
+    realm_users: [],
+});
+
+set_global('$', global.make_zjquery());
+
+set_global('channel', {
+});
+
+add_dependencies({
+    people: 'js/people.js',
+    stream_data: 'js/stream_data.js',
+});
+
+set_global('blueslip', {});
+
+var integration_bot_widget = require('js/integration_bot_widget.js');
+
+(function test_set_integration_bot_url() {
+    var success_callback;
+    var error_callback;
+    var posted;
+
+    var form_data = {
+        append: function (field, val) {
+            form_data[field] = val;
+        },
+    };
+
+    set_global('csrf_token', 'token-stub');
```

Comment:
maybe we should move this to reside outside the testing function and say something like
`set_global('csrf_token', 'csrf-token-stub');` around L9 or L10